x-pack/qa/core-rest-tests-with-security/src/yamlRestTest/resources/rest-api-spec/test/synonyms_privileges/10_synonyms_with_privileges.yml

```diff
@@ -0,0 +1,111 @@
+setup:
+  - skip:
+      features: headers
+      version: " - 8.9.99"
+      reason: Introduced in 8.10.0
+
+  - do:
+      security.put_user:
+        username: "synonyms-user"
+        body:
+          password: "synonyms-user-password"
+          roles : [ "synonyms-role" ]
+
+  - do:
+      security.put_role:
+        name: "synonyms-role"
+        body:
+          cluster: ["manage_search_synonyms"]
+
+  - do:
+      security.put_user:
+        username: "non-synonyms-user"
+        body:
+          password: "non-synonyms-user-password"
+          roles : [ "non-synonyms-role" ]
+
+  - do:
+      security.put_role:
+        name: "non-synonyms-role"
+        body:
+          indices:
+            - names: ["*"]
+              privileges: [ "manage", "write", "read" ]
```

Comment:
Nit: I think these user non-synonyms user and role can be removed since we now split the tests into 2 files? The same goes for the `teardown` part.

Reply:
🤦 Removing them, thanks for the catch!